OneCommunityGlobal/HighestGoodNetworkApp
# Yi - Dashboard: refine editable info modal tooltips text

# Description
<img width="811" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59299011/e1888875-6850-48f4-8e73-c21ed10e9dde">

## Related PRS (if any):
This frontend PR is related to the backend dev.

## Main changes explained:
Change the tooltips text from "Click for user class information" to "Click for information about this"
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. login
5. go to dashboard
6. verify all tooltips in dashboard

## Screenshots or videos of changes

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59299011/b4c375dc-79fc-4899-8558-e2550bb10f15
